docs/details/interface.md

```diff
@@ -17,9 +17,15 @@ The main menu is part of the Grafana framework and is visible on every page.
 |:----------------------------------:|----------------------|-------------------------------
 | {{icon.percona}}                   | Home                 | Link to home dashboard.
 | <i class="uil uil-search"></i>     | Search               | Search dashboards by name.
-| <i class="uil uil-plus"></i>       | Create               | Create dashboards or [folders][Folders], import dashboards.
-| <i class="uil uil-apps"></i>       | Dashboards           | Manage dashboards, create playlists, manage snapshots.
-| {{icon.dashboards}}                | PMM Dashboards       | Replacement for [shortcut menu](#shortcut-menu).
+| <i class="uil uil-star"></i>       | Starred              | Mark your favorite dashboards.
+| <i class="uil uil-apps"></i>       | Dashboards           | Create dashboards or [folders][Folders], manage dashboards, import dashboards, create playlists, manage snapshots.
+| ![!image](../_images/os-dashboard.png)      | Operating System (OS)    | Operating System dashboard
+| ![!image](../_images/mysql-dashboard.png)    | MySQL      | MySQL dashboard
+| ![!image](../_images/mongo-dashboard.png)    | MongoDB      | MongoDB dashboard
+| ![!image](../_images/postresql-dashboard.png)    | PostgreSQL     | PostgreSQL dashboard
+| ![!image](../_images/proxysql-dashboard.png)    | ProxySQL     | ProxySQL dashboard
+| ![!image](../_images/haproxy-dashboard.png)    | HAproxy     | HAproxy dashboard
+| ![!image](../_images/qan-dashboard.png)| Query Analytics (QAN) | Query Analytics
 | <i class="uil uil-compass"></i>    | Explore              | Run queries with [PromQL].
 | <i class="uil uil-bell"></i>       | Alerting             | Alerting, [Integrated Alerting](../using/alerting.md), Alert Rules, Notification Channels.
```

Comment:
We renamed Alerting as Percona Alerting. 
So, I'd suggest to use new name and we may re-write this part like:
Percona Alerting; create new alerts and manage your alert rules.

Reply:
@catalinaadam Could you please make this change as there is a link to Integrated Alerting? Could you please provide the correct link.